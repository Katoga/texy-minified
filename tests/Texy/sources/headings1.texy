
A First Level Header
####################

A Second Level Header
*********************

A Third Level Header
====================

A Fourth Level Header
---------------------

#### Největší titulek

### střední titulek

## malý titulek
