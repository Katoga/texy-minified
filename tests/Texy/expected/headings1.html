<h1>A First Level Header</h1>

<h2>A Second Level Header</h2>

<h3>A Third Level Header</h3>

<h4>A Fourth Level Header</h4>

<h1>Největší titulek</h1>

<h2>střední titulek</h2>

<h3>malý titulek</h3>
